cli/tests/services.bats

```diff
@@ -585,13 +565,16 @@ EOF
   for command in "${commands[@]}"; do
     echo "Testing: flox services $command"
     command="$command" run "$FLOX_BIN" activate -- bash <(cat <<EOF
-      source "${TESTS_DIR}/services/register_cleanup.sh"
 
       [ ! -e "\$_FLOX_SERVICES_SOCKET" ] || exit 2
 
       "$FLOX_BIN" services "$command"
 EOF
 )
+
+    # give the watchdog a chance to clean up the services before the next iteration
+    sleep 0.5
+
```

Comment:
nit: in `teardown` we wait 0.1[s], while here we wait 0.5[s]
question: is there anything we could poll for instead?

Reply:
Polled in https://github.com/flox/flox/pull/2018/commits/43f913c45b00d22e63804e3ce90eed3c274981eb